src/main/java/us/jonathans/view/JMancalaPanel.java

```diff
@@ -197,7 +198,13 @@ public void createleaderboardButton() {
         getLeaderboardButton.addActionListener(
                 evt -> {
                     if(evt.getSource().equals(getLeaderboardButton)) {
-                        getLeaderboardController.execute();
+                        try {
+                            getLeaderboardController.execute();
+                        } catch (IOException e) {
+                            throw new RuntimeException(e);
+                        } catch (InterruptedException e) {
+                            throw new RuntimeException(e);
+                        }
```

Comment:
Get rid of this try catch block. If you move the error handling lower you should't need it.